### PR TITLE
Makes the Collection tabs into links so we can show Collection Summary info

### PR DESCRIPTION
**Pull request by sdellis**

### CODE REVIEW

File: app/assets/javascripts/tabs.js [hunk 1]
```diff
@@ -24,36 +24,45 @@ $( document ).ready(function() {
...
-    }
+  $(".lux-tabs-container li").click(function() {
+    let temp_id = $(this).attr('id').split("-");
+    temp_id.pop()
+    window.location.hash = temp_id.join("-");
+    console.log($(this).attr('id'))
+    window.location.reload()
   });
+
+  // $(".lux-tabs-container li").click(function( event ) {
```
Comment (tpendragon) [inline app/assets/javascripts/tabs.js:38]: Can you delete this instead of commenting?

File: app/views/catalog/_show_tabs.html.erb [hunk 2]
```diff
@@ -1,26 +1,26 @@
 <ul class="lux-tabs-container" role="tablist" aria-label="Collection Notes">
   <li id="summary-tab" class="lux-tabs-item" role='tab'>
-    <a class='nav-link' href='#summary'>
-      Collection Overview
+    <a class='nav-link' href='<%= solr_document_path(document.http_safe_ead) %>'>
```
Comment (tpendragon) [inline app/views/catalog/_show_tabs.html.erb:3]: What's `http_safe_ead`? Did `document.id` not work?

File: app/assets/javascripts/tabs.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 $( document ).ready(function() {
   const url = new URL(window.location);
-
+  $(".lux-tabs-container li").removeClass( "active" );
```
Comment (tpendragon) [inline app/assets/javascripts/tabs.js:3]: A weird thing happens when I press "back" in the browser - the anchor sticks around as #access but on the component. Maybe a fluke?

**Review by tpendragon — CHANGES_REQUESTED** [review]

File: app/views/catalog/_show_tabs.html.erb (hunk 2 above)
Comment (sdellis) [reply app/views/catalog/_show_tabs.html.erb:3]: I needed the Collection URL, and that was the way I found I could get it. The Document.id is the component URL if you're on a component.

File: app/assets/javascripts/tabs.js (hunk 3 above)
Comment (sdellis) [reply app/assets/javascripts/tabs.js:3]: @tpendragon I think I fixed this in my latest push. LMK if that works for you.

File: app/assets/javascripts/tabs.js (hunk 1 above)
Comment (sdellis) [reply app/assets/javascripts/tabs.js]: Yes. I added a permalink to the old code for future reference.

File: app/assets/javascripts/tabs.js (hunk 3 above)
Comment (tpendragon) [reply app/assets/javascripts/tabs.js:3]: @sdellis The back button thing still happens for me. https://pulfalight-staging.princeton.edu/catalog/C0159_c000001 Click "access and use" and then hit the back button. Maybe we'll have to just not render those tabs at all unless you're on a collection component?

Comment (sdellis) [reply app/assets/javascripts/tabs.js:3]: @tpendragon I think I finally got it with this latest commit.🤞  Thanks for catching that.

Comment (tpendragon) [reply app/assets/javascripts/tabs.js:3]: Yup, that seems to work. PR just needs rebased tomorrow and we're good.

**Review by tpendragon — APPROVED** [review]

**Pull request merged**